### PR TITLE
Update road_atlanta2018.ini

### PR DESCRIPTION
**Pull request by Vannipo**
New [MATERIAL_ADJUSTMENT_14], replaced all conditions NIGHT_SMOOTH, NIGHT_SMOOTH_HEATING to SMOOTH_SUN_A

### CODE REVIEW

File: config/tracks/road_atlanta2018.ini [hunk 1]
```diff
@@ -1,5 +1,8 @@
- ; config was prepared automatically. source:
- ; https://github.com/ac-custom-shaders-patch/acc-extension-config/blob/master/config/tracks/road_atlanta2018.ini
+; config was prepared automatically. source:
+; https://github.com/ac-custom-shaders-patch/acc-extension-config/blob/master/config/tracks/road_atlanta2018.ini
+; config was prepared automatically. source:
+; https://github.com/ac-custom-shaders-patch/acc-extension-config/blob/master/config/tracks/road_atlanta2018.ini
+; Vannipo: New [MATERIAL_ADJUSTMENT_14], replaced all conditions NIGHT_SMOOTH, NIGHT_SMOOTH_HEATING to SMOOTH_SUN_A
```
Comment (Marocco2) [inline config/tracks/road_atlanta2018.ini:5]: Delete all those lines above

File: config/tracks/road_atlanta2018.ini [hunk 2]
```diff
@@ -1,5 +1,8 @@
- ; config was prepared automatically. source:
- ; https://github.com/ac-custom-shaders-patch/acc-extension-config/blob/master/config/tracks/road_atlanta2018.ini
+; config was prepared automatically. source:
+; https://github.com/ac-custom-shaders-patch/acc-extension-config/blob/master/config/tracks/road_atlanta2018.ini
+; config was prepared automatically. source:
+; https://github.com/ac-custom-shaders-patch/acc-extension-config/blob/master/config/tracks/road_atlanta2018.ini
+; Vannipo: New [MATERIAL_ADJUSTMENT_14], replaced all conditions NIGHT_SMOOTH, NIGHT_SMOOTH_HEATING to SMOOTH_SUN_A
 [ABOUT]
 AUTHOR = Blamer
 VERSION = 2.0.2
```
Comment (Marocco2) [inline config/tracks/road_atlanta2018.ini:8]: Update version number and add notes in NOTES

**Review by Marocco2 — CHANGES_REQUESTED** [review]
Check my comments and you're good to go

File: config/tracks/road_atlanta2018.ini (hunk 1 above)
Comment (Vannipo) [reply config/tracks/road_atlanta2018.ini]: Done

File: config/tracks/road_atlanta2018.ini (hunk 2 above)
Comment (Vannipo) [reply config/tracks/road_atlanta2018.ini]: Done. Hopefully correct

**Pull request merged**